app.js

```diff
@@ -0,0 +1,188 @@
+  await client.connect();
+
+  // Provide the name of the database and collection you want to use.
+  // If the database and/or collection do not exist, the driver and Atlas
+  // will create them automatically when you first write data.
+  const dbName = "myDatabase";
+  const collectionName = "recipes";
+
+  // Create references to the database and collection in order to run
+  // operations on them.
+  const database = client.db(dbName);
+  const collection = database.collection(collectionName);
+
+  /*
+   *  *** INSERT DOCUMENTS ***
+   *
+   * You can insert individual documents using collection.insert().
+   * In this example, we're going to create four documents and then
+   * insert them all in one call with collection.insertMany().
+   */
+
+  const recipes = [
+    {
+      name: "elotes",
+      ingredients: [
+        "corn",
+        "mayonnaise",
+        "cotija cheese",
+        "sour cream",
+        "lime",
+      ],
+      prepTimeInMinutes: 35,
+    },
+    {
+      name: "loco moco",
+      ingredients: [
+        "ground beef",
+        "butter",
+        "onion",
+        "egg",
+        "bread bun",
+        "mushrooms",
+      ],
+      prepTimeInMinutes: 54,
+    },
+    {
+      name: "patatas bravas",
+      ingredients: [
+        "potato",
+        "tomato",
+        "olive oil",
+        "onion",
+        "garlic",
+        "paprika",
+      ],
+      prepTimeInMinutes: 80,
+    },
+    {
+      name: "fried rice",
+      ingredients: [
+        "rice",
+        "soy sauce",
+        "egg",
+        "onion",
+        "pea",
+        "carrot",
+        "sesame oil",
+      ],
+      prepTimeInMinutes: 40,
+    },
+  ];
+
+  collection
+    .insertMany(recipes)
+    .then(result =>
+      console.log(`${result.insertedCount} documents successfully inserted.`),
+    )
+    .catch(err =>
+      console.error(
+        `Something went wrong trying to insert the new documents: ${err}`,
+      ),
+    );
+
+  /*
+   * *** FIND DOCUMENTS ***
+   *
+   * Now that we have data in Atlas, we can read it. To retrieve all of
+   * the data in a collection, we call Find() with an empty filter.
+   * The Builders class is very helpful when building complex
+   * filters, and is used here to show its most basic use.
+   */
+
+  const findQuery = { prepTimeInMinutes: { $lt: 45 } };
+
+  try {
+    const cursor = await collection.find(findQuery).sort({ name: 1 });
+    await cursor.forEach(recipe =>
+      console.log(
+        `${recipe.name} has ${recipe.ingredients.length} ingredients and takes ${recipe.prepTimeInMinutes} minutes to make`,
+      ),
+    );
+  } catch (err) {
+    console.error(`Something went wrong trying to find the documents: ${err}`);
+  }
+
+  // We can also find a single document. Let's find the first document
+  // that has the string "potato" in the ingredients list.
+  const findOneQuery = { ingredients: "potato" };
+
+  try {
+    const findOneResult = await collection.findOne(findOneQuery);
```

Comment:
Do we want to declare victory by displaying the `findOneResult` in a console log?

Reply:
I had the same question. I was mirroring the C# starter project which does not print it:
https://github.com/mongodb-university/atlas_starter_dotnet/blob/master/AtlasStarter/Program.cs#L110
Happy to print it out though.

Reply:
Yeah - let's do it!  I'll PR the .NET to reflect the same.  Thanks for noting!

Reply:
https://github.com/mongodb-university/atlas_starter_dotnet/pull/5